compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/BUILD.bazel

```diff
@@ -0,0 +1,86 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+load("//build_tools/bazel:build_defs.oss.bzl", "iree_compiler_cc_library", "iree_gentbl_cc_library")
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = ["layering_check"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+iree_gentbl_cc_library(
+    name = "PassesIncGen",
+    tbl_outs = [
+        (
+            ["--gen-pass-decls"],
+            "Passes.h.inc",
+        ),
+    ],
+    tblgen = "@llvm-project//mlir:mlir-tblgen",
+    td_file = "Passes.td",
+    deps = [
+        "@llvm-project//mlir:PassBaseTdFiles",
+    ],
+)
+
+iree_compiler_cc_library(
+    name = "Transforms",
+    srcs = [
+        "ConvertConv2DToWinograd.cpp",
+        "ConvertToLoops.cpp",
+        "ForeachThreadToAsync.cpp",
+        "PadContractionToBlockSize.cpp",
+        "PassDetail.h",
+        "Passes.cpp",
+        "SplitReduction.cpp",
+        "TileAndDecomposeAttention.cpp",
+        "TileAndDecomposeWinogradPass.cpp",
+    ],
+    hdrs = [
+        "Passes.h",
+        "Passes.h.inc",
+        "Transforms.h",
+    ],
+    deps = [
+        ":PassesIncGen",
+        "//compiler/src/iree/compiler/Dialect/LinalgExt/IR",
+        "//compiler/src/iree/compiler/Dialect/LinalgExt/Utils",
+        "//llvm-external-projects/iree-dialects:IREEInputDialect",
+        "@llvm-project//llvm:Support",
+        "@llvm-project//mlir:AffineDialect",
+        "@llvm-project//mlir:AffineUtils",
+        "@llvm-project//mlir:ArithDialect",
+        "@llvm-project//mlir:ArithUtils",
+        "@llvm-project//mlir:AsyncDialect",
```

Comment:
I think `Async` dialect was pulled in for some experimentation. We should be able to just drop it.